internal/testutils/cmd.go

```diff
@@ -0,0 +1,29 @@
+package testutils
+
+import (
+	"log"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+// RunWithTiming runs the given command while logging its duration with the provided message.
+func RunWithTiming(msg string, cmd *exec.Cmd) error {
+	log.Printf("%s: %s", msg, strings.Join(cmd.Args, " "))
+	start := time.Now()
+	err := cmd.Run()
+	duration := time.Since(start)
+	log.Printf("Finished in %.3fs", duration.Seconds())
+	return err
+}
+
+// CombinedOutputWithTiming runs the given command while logging its duration with the provided message
+// and returns its combined standard output and standard error.
+func CombinedOutputWithTiming(msg string, cmd *exec.Cmd) ([]byte, error) {
```

Comment:
Maybe just use options and reuse the same code of `RunWithTiming`?

Or an internal implementation that takes a function that does the run.

Reply:
Can we do that once the duplicated code becomes less trivial? For now it's very small and IMO does not warrant the added complexity of adding options or adding a level of indirection.